README.md

```diff
@@ -142,20 +142,21 @@ optional arguments:
 In the folder where you are running Commander, create a file called config.json which defines the
 API server and the email address.
 
-For customers hosted in the US region, set the US endpoint:
-```shell script
-{
-  "server": "https://keepersecurity.com/api/v2/"
-}
-```
+Commander defaults to the US region, so customers hosted in other regions will need to specify a server in the config.
 
-For customers hosted in the EU region, set the EU endpoint:
+####Example config.json for EU region
```

Comment:
Does not look correct in the rendered version
![image](https://user-images.githubusercontent.com/187306/113163794-5a201880-91f5-11eb-9153-24aeb58c2617.png)